src/Etg.Yams.Client/YamsClient.cs

```diff
@@ -91,16 +91,21 @@ public void Dispose()
             }
             while (true)
             {
-                Trace.TraceInformation("Waiting for an exit message from Yams..".FormatMessage(_options));
+                Trace.TraceInformation(FormatMessage("Waiting for an exit message from Yams..", _options));
                 string msg = await _exitConnection.ReadMessage();
                 if (msg == "[EXIT]")
                 {
-                    Trace.TraceInformation("Exit request received from Yams".FormatMessage(_options));
+                    Trace.TraceInformation(FormatMessage("Exit request received from Yams", _options));
                     ExitMessageReceived?.Invoke(this, EventArgs.Empty);
                     break;
                 }
-                Trace.TraceError($"Unexpected message received from Yams: {msg}, Expected [EXIT]".FormatMessage(_options));
+                Trace.TraceError(FormatMessage($"Unexpected message received from Yams: {msg}, Expected [EXIT]", _options));
             }
         }
+
+        private static string FormatMessage(string message, YamsClientOptions options)
```

Comment:
Why pass in options? It's a field in the class it would make things a bit more readable to access it directly from the field.

Reply:
Fair enough, fixing it now